chainerx_cc/chainerx/routines/indexing.cc

```diff
@@ -308,7 +308,8 @@ std::vector<Array> Nonzero(const Array& a) {
     if (count_nonzero > 0) {
         Array addat_indices = Where(is_nonzero, Maximum(Cumsum(is_nonzero) - 1, 0), 0);
         Array indices = Where(is_nonzero, Arange(a.GetTotalSize(), Dtype::kInt64), 0);
-        a.device().backend().CallKernel<AddAtKernel>(raw_index, std::move(addat_indices), 0, std::move(indices), raw_index);
+        a.device().backend().CallKernel<AddAtKernel>(
+             raw_index, std::move(addat_indices), 0, std::move(indices), raw_index, IndexBoundsMode::kDefault);
```

Comment:
Ditto regarding not using default.

Reply:
Here the default value is different for cuda, and native, doing a check implies to have another place defining the default behavior which can lead to inconsistencies if it is changed.

Reply:
You're right, let's leave it as is.